common/src/main/java/com/datatorrent/common/util/PubSubWebSocketClient.java

```diff
@@ -376,7 +385,8 @@ public void unsubscribe(String topic) throws IOException
    * @return subscribe num subscriber message.
    * @throws IOException
    */
-  public static <T> String constructSubscribeNumSubscribersMessage(String topic, PubSubMessageCodec<T> codec) throws IOException
+  public static <T> String constructSubscribeNumSubscribersMessage(String topic, PubSubMessageCodec<T> codec) throws
+      IOException
```

Comment:
No it is very hard. You can take up the changes in engine module and try it out .